_pages/en_US/troubleshooting.txt

```diff
@@ -609,3 +655,18 @@ This can occur after a CTRTransfer or region change on Old 3DS / 2DS. You will n
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+
+<summary><u>Turning off Parental Controls</u></summary>
```

Comment:
if you know the pin you can just disable it without the masterkey, this is nowhere mentioned and might be misleading

Reply:
It kind of is mentioned, as it says do the mkey steps "if you do not know the PIN and it is stopping you from accessing the device's settings"

Reply:
they fixed it in another line, when I submitted the review that text wasnt there. (this can be resolved)